docs/xxlegacyxlogoption.md

```diff
@@ -24,14 +24,21 @@
 
 # -XX:[+|-]LegacyXlogOption
 
-Controls processing of the [`-Xlog`](xlog.md) option. When `-XX:-LegacyXlogOption` is set, which is the default, the `-Xlog` option is not recognized.
-When `-XX:+LegacyXlogOption` is set, the `-Xlog` option is equivalent to the [`-Xsyslog`](xsyslog.md) option. 
+Controls processing of the `-Xlog` option.
 
```

Comment:
I suggest we start with the following content (or similar), then just have the table. I don't think we need to talk about what the "-" option does as that's already explained in the -Xlog topic (and if there's a change in future, we'd have to update two topics instead of one), plus the only users who need to know about this legacy option are those who want to turn it on.

> Controls the behavior of the -Xlog option. In Eclipse OpenJ9 0.24.0, the -Xlog option changed for better compatibility with the reference implementation. The old -Xlog behavior was transferred to the new -Xsyslog option. If you cannot use -Xsyslog for some reason, you can make -Xlog behave as it did in earlier releases by specifying -XX:+LegacyXlogOption.

Reply:
@pshipton - please see Esther's comment as an approach for explaining what we are achieving here. If we went with this, I'd have to update some of the other topics to adopt this. Would you support this?

In this case, we'd not talk too much about the `-XX` option in the `-Xlog` topic other than to reference it as a means of changing `-Xlog` behavior.

Reply:
Works for me.